src/datascience-ui/data-explorer/data-wrangler/controls/toolbar.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { DataWranglerCommands } from '../../../../client/datascience/data-viewing/data-wrangler/types';
+
+interface IProps {
+    handleRefreshRequest(): void;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    submitCommand(data: { command: string; args: any }): void;
+    onToggleFilter(): void;
+}
+
+export class Toolbar extends React.PureComponent<IProps> {
+    render() {
+        return (
+            <div
+                style={{
+                    borderBottom: '1px solid var(--vscode-editor-inactiveSelectionBackground)',
+                    paddingTop: '4px',
+                    paddingBottom: '6px',
+                    paddingLeft: '5px',
+                    display: 'flex',
+                    flexDirection: 'row',
+                    fontFamily: 'var(--vscode-font-family)',
+                    fontSize: 'var(--vscode-font-size)',
+                    // fontWeight: 'var(--vscode-font-weight)',
+                    justifyContent: 'start'
+                }}
+            >
+                <div
+                    style={{ paddingRight: '15px', display: 'inline-block', cursor: 'pointer' }}
+                    onClick={() => this.props.submitCommand({ command: DataWranglerCommands.ExportToCsv, args: null })}
+                >
+                    <div
+                        className="codicon codicon-export codicon-button"
+                        style={{ verticalAlign: 'middle' }}
+                        title="Export to CSV"
+                    />
+                    <span style={{ verticalAlign: 'middle', paddingLeft: '4px', paddingBottom: '4px' }}>
+                        Export CSV
+                    </span>
+                </div>
+                <div
+                    style={{ paddingRight: '15px', display: 'inline-block', cursor: 'pointer' }}
+                    onClick={() =>
+                        this.props.submitCommand({ command: DataWranglerCommands.ExportToPythonScript, args: null })
+                    }
+                >
+                    <div
+                        className="codicon codicon-go-to-file codicon-button"
+                        style={{ verticalAlign: 'middle' }}
+                        title="Open as Python script"
+                    />
+                    <span style={{ verticalAlign: 'middle', paddingLeft: '4px', paddingBottom: '4px' }}>
+                        Open as Python script
+                    </span>
+                    {/* <div className="codicon codicon-notebook codicon-button" title="Open in Notebook" /> */}
+                </div>
+                {/* <div
```

Comment:
Why is a bunch of this commented out? Do we intend to enable this in the future?

Reply:
still wasn't sure if they were going to be included or not but got confirmation and removed them now